internal/integration/fixtures/extra-volume-mounts.yaml

```
@@ -0,0 +1,30 @@
+steps:
+  - label: ":wave:"
+    agents:
+      queue: {{.queue}}
+    env:
+      BUILDKITE_GIT_CLONE_MIRROR_FLAGS: --mirror
+      BUILDKITE_GIT_MIRRORS_PATH: /tmp/buildkite-git-mirrors
+    plugins:
+      - kubernetes:
+          podSpec:
+            containers:
+              - image: alpine:latest
+                command: [ash]
+                args:
+                  -  -c
+                  - "echo 'volume mounted' > /tmp/buildkite-git-mirrors/foo.txt"
```

```suggestion
                  - '"echo volume mounted > /tmp/buildkite-git-mirrors/foo.txt"'
```
Turns out, when the YAML is parsed, the double quotes are dropped. But they are needed so that `ash -c` receives a single argument.

`echo` is perfectly capable of taking multiple arguments, so the single quotes aren't necessary, i.e., `echo 'hello world'` and `echo hello world` produce the same output.

So, I think if we remove the single quotes from inside the string, we can use single quotes to escape the double quotes that we need to be in the string.

---

thank you @triarius for spending time on this.
I have pushed with the recommended syntax.
besides that, I'm not sure to understand what is blocking you from pushing to my downstream repo, I checked the PR and it has "Allow edits by maintainers" checked. I've also added you as a collaborator in the meantime.

---

Sorry, my message was a bit unclear:
1. I did have the ability to push to your downstream repo.
2. I did push commits to it.
3. I force pushed those commits away.
4. Adding me as a collaborator was not necessary, as you always had "Allow edits by maintainers" checked

I only brought it up because I think it could be a security risk for you to allow me to push commits to your downstream repo without explicitly allowing me to. But now that you added me as a collaborator, this point is moot.

BTW, feel free to remove me.